File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
import java.util.ArrayList;
import java.util.*;
class Solution {
    public String solution(String my_string, int num1, int num2) {
        String answer = "";

        ArrayList<Character> list=new ArrayList<>();
        for(char x:my_string.toCharArray()){
            list.add(x);
        }
        for(int i=0; i<list.size(); i++){
            if(i==num1){
                answer+=list.get(num2);
            }else if(i==num2){
               answer+=list.get(num1);
            }else
                answer+=list.get(i);
        }

        return answer;
    }
}

File: 프로그래머스/unrated/120895. 인덱스 바꾸기/인덱스 바꾸기.swift
import Foundation

func solution(_ my_string:String, _ num1:Int, _ num2:Int) -> String {
    
    var a = my_string.index(my_string.startIndex, offsetBy: num1)
    var b = my_string.index(my_string.startIndex, offsetBy: num2)
    
    var answer = my_string
    
    let ch1 = my_string[a]
    let ch2 = my_string[b]
    
    answer.replaceSubrange(a...a, with: String(ch2))
    answer.replaceSubrange(b...b, with: String(ch1))
    return answer
}
